doc/word.xml

```diff
@@ -0,0 +1,104 @@
+#############################################################################
+##
+#W  word.xml
+#Y  Copyright (C) 2020                                      Maria Tsalakou
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+
+<#GAPDoc Label="WordToString" >
+<ManSection>
+  <Oper Name = "WordToString" Arg = "alphabet, word"
+    Label="for a string and a list"/>
+  <Returns>
+    The <A>word</A> in the form of a string.
+  </Returns>
+  <Description>
+    Returns the word <A>word</A> that is given in the form of a list, in 
```

Comment:
I would recommend adding a comment here on the fact that `alphabet` has to be a string. I don't think the `Label` that you've added works for all platforms; it doesn't seem to appear anywhere in the HTML documentation!